File: Very Easy/back_to_home.py
# https://edabit.com/challenge/F3M4PhqC4JdX28Qmx

def back_to_home(directions):
    if not directions.count("N")>=2:
        return False
    elif not directions.count("S")>=2:
        return False
    elif not directions.count("E")>=2:
        return False
    elif not directions.count("W")>=2:
        return False
    return True

def back_to_home(directions):
    n = directions.count('N')
    e = directions.count('E')
    s = directions.count('S')
    w = directions.count('W')

    if n == s and e == w:
        return True
    return False

print(back_to_home("NNSSEEEWWWEW"), True)

File: javascript/Easy/backToHome.js
// https://edabit.com/challenge/MQvP7E5sNRswScHpn


function backToHome(directions){
  const final  = ['N','E','S',"W"];
  for (let i = 0; i < final.length; i++) {
     const letter=  new RegExp(`${final[i]}`,'g')
     const test  = directions.match(letter)
     if(test==null|| test.length%2!=0){
         return false
     }
    } 
    return true
    //    console.log(final)   
}

console.log(backToHome("NNNN"), false)
console.log(backToHome("NENESSWW"), true)
console.log(backToHome("NEESSW"), false)
console.log(backToHome("EEWE"), false)
console.log(backToHome("NNSSEEEWWWEW"), true)